alpine-nix-rails-nginx/Dockerfile

```diff
@@ -0,0 +1,53 @@
+# dkubb/alpine-nix-rails-nginx
+
+FROM dkubb/alpine-nix:latest
+MAINTAINER Dan Kubb <dkubb@fastmail.com>
+
+EXPOSE 80
+
+# Configure nix
+COPY config.nix /root/.nixpkgs/config.nix
+
+# Install system dependencies
+RUN nix-env --install --fallback \
+    acl-2.2.52 \
```

Comment:
I'm not actually sure either. I mean, you can just go to the [nixpkgs](https://github.com/NixOS/nixpkgs) repo and search for the packages.

Or you can do:

```bash
nix-channel --update
nix-env -qaP '.*acl.*'
```